Changelog.md

```diff
@@ -1,5 +1,32 @@
 ## Gazebo 11
 
+## Gazebo 11.10.0 (2022-01-12)
+
+1. Use boost/bind/bind.hpp to fix warnings on Arch Linux
+    * [Pull request #3156](https://github.com/ignitionrobotics/gazebo/pull/3156)
+    * Inspired by a contribution from Alex Dewar <alex.dewar@gmx.co.uk>
+
+1. Fix focal builds: use python3 with check_test_ran.py
+    * [Pull request #3155](https://github.com/ignitionrobotics/gazebo/pull/3155)
+
+1. Load model plugins even after sensor timeout
+    * [Pull request #3154](https://github.com/ignitionrobotics/gazebo/pull/3154)
+
+1. CMake exports: remove -std=c++11 flag
+    * [Pull request #3050](https://github.com/ignitionrobotics/gazebo/pull/3050)
+    * A contribution from Guilhem Saurel <guilhem.saurel@laas.fr>
+
+1. Create github action ci
+    * [Pull request #3049](https://github.com/ignitionrobotics/gazebo/pull/3049)
+    * A contribution from Tal Regev <tal.regev@gmail.com>
+
+1. Fix Windows build when using vcpkg
+    * [Pull request #3132](https://github.com/ignitionrobotics/gazebo/pull/3132)
+    * A contribution from Akash Munagala <akash.munagala@gmail.com>
+
+1. Point light shadows
+    * [Pull request #3051](https://github.com/ignitionrobotics/gazebo/pull/3051)
+
```

Comment:
@scpeters The links to the pull requests should be to the https://github.com/osrf/gazebo/ repo and not https://github.com/ignitionrobotics/gazebo/, right ?

Reply:
yes! thanks for catching that. I've included a fix in https://github.com/osrf/gazebo/pull/3160

Reply:
Thanks for the quick fix :+1: